bin/check-smart.rb

```diff
@@ -102,6 +111,12 @@ class CheckSMART < Sensu::Plugin::Check::CLI
          proc: proc(&:to_sym),
          default: :unknown
 
+  option :no_smart_capable_disks,
+         long: '--no-smart-capable-disks EXIT_CODE',
+         description: 'Exit code when there are no SMART capable disks (ok, warn, critical, unknown)',
+         proc: proc(&:to_sym),
+         default: :unknown
+
```

Comment:
we should restrict it to the known available ones like this:
```ruby
option :no_smart_capable_disks,
       long: '--no-smart-capable-disks EXIT_CODE',
       description: 'Exit code when there are no SMART capable disks (ok, warn, critical, unknown)',
       proc: proc(&:to_sym),
￼       default: unknown, # with doing a `proc(&:to_sym)` you don't need to make it a symbol here. Hide complexity from users where possible.
       in: %w(unknown warning critical warning ok)
```

Reply:
I'm happy to change this, but to be consistant with the 'smart_incapable_disks' option, do you want that updated in the same fassion?

Reply:
yes please.

Reply:
The only way I was able to get this to work was to use:
```
         :default => :unknown,
         :in => %i(unknown ok warn critical)
```

If I didnt use symbols, it refused to find the supplied value in the list.

Reply:
ok, makes sense when I think about it.